how-to/customize-workspace/client/src/framework/shapes/headless-shapes.ts

```diff
@@ -1,4 +1,9 @@
 import type { ModuleList } from "./module-shapes";
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
+/**
+ * The headless provider supports a list of modules. Right now the url must be to a
+ * html page as we will be launching a window after the bootstrapping process has run. A js extension will log an error.
+ * The data setting of the module can be a WindowsOptions object with settings (our defaults should be enough if you want
+ * a hidden window to run logic in the background)
+ */ // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface HeadlessProviderOptions extends ModuleList {}
```

Comment:
Maybe change this to `export type HeadlessProviderOptions = ModuleList;` to match similar objects, then we can remove the eslint warning.

Reply:
Updated